docs/blog/2019-07-03-themes-are-stable/index.md

```diff
@@ -0,0 +1,74 @@
+---
+title: Introducing Gatsby Themes
+date: 2019-07-03
+author: Chris Biscardi
+excerpt: "Gatsby themes go stable and what that means"
+tags: ["themes"]
+---
+
+Let's start with the question you're probably asking: When are Gatsby Themes going to be stable?
```

Comment:
Maybe make this more matter of fact (a lot of people -- non-early-adopters-- are probably reading about themes for the first time)

"We've been working on the "themes" feature of Gatsby since [late last year](/blog/2018-11-11-introducing-gatsby-themes/), and we're excited to announce today that we've released themes as stable."?

Reply:
```suggestion
## What are Gatsby themes?

Using a Gatsby theme, all of your default configuration (shared functionality, data sourcing, design) is abstracted out of your site, and into an installable package.

This means that the configuration and functionality isn’t directly written into your project, but rather versioned, centrally managed, and installed as a dependency. You can seamlessly update a theme, compose themes together, and even swap out one compatible theme for another.
```

Reply:
Not sure if you agree with linking people over to the docs from here, but might make sense to give people who are less familiar with themes a natural escape hatch. Diving into the nitty gritty will make sense and be appropriate for people who have been following experimental themes, but I can see some people getting lost.

Reply:
FWIW, my generic go-to is *Say hello to...*